harfbuzz/fix-cmake.patch

```diff
@@ -0,0 +1,14 @@
+diff -ruN original/CMakeLists.txt new/CMakeLists.txt
+--- original/CMakeLists.txt	2023-11-11 15:08:45.000000000 +0100
++++ new/CMakeLists.txt	2024-03-18 22:57:40.191313381 +0100
+@@ -116,7 +116,9 @@
+   find_package(Threads)
+   if (CMAKE_USE_PTHREADS_INIT)
+     add_definitions("-DHAVE_PTHREAD")
+-    list(APPEND THIRD_PARTY_LIBS Threads::Threads)
++    if (NOT PSP)
++      list(APPEND THIRD_PARTY_LIBS Threads::Threads)
++    endif()
+     list(APPEND PC_LIBS_PRIV -pthread)
```

Comment:
I just did a test and this part also need to not be for PSP.
```suggestion
+    if (NOT PSP)
+      list(APPEND THIRD_PARTY_LIBS Threads::Threads)
+      ist(APPEND PC_LIBS_PRIV -pthread)
+    endif()
-     list(APPEND PC_LIBS_PRIV -pthread)
```
I did this suggestion by hand, so give me a second to test it

Reply:
I updated the patch.